app/models/solution.rb

```diff
@@ -44,6 +44,14 @@ def self.not_legacy
     where("solutions.created_at >= ?", Exercism::V2_MIGRATED_AT)
   end
 
+  def display_published_at
```

Comment:
Thanks for solving this. Could you do this using the tertiary operator to keep things a bit slimmer pls?